test/lsp_test_runner.cc

```diff
@@ -267,7 +261,7 @@ void testDocumentSymbols(LSPWrapper &lspWrapper, Expectations &test, int &nextId
 
 TEST_CASE("LSPTest") {
     /** The path to the test Ruby files on disk */
-    UnorderedSet<std::string> filenames;
```

Comment:
This set introduced nondeterministic file ID ordering, which apparently isn't a problem on existing tests but caused issues with some of the package tests. I converted it into a vector.